### ISSUE TITLE
Data transfer rate is hard to read and understand (accessibility issue)

### ISSUE THREAD

**Issue by rebelcode-xr** [transmission/transmission#4774]
### What is the issue?

Thank you for the new 4.0 version and your hard work! 🙏

I use Transmission on Mac and the new transfer speed color for upload (white text on light green background) is pretty hard to read:
<img width="104" alt="image" src="https://user-images.githubusercontent.com/63522951/217856560-341f6a91-8153-444c-8b77-9ec8337e70ac.png">
I suggest to put a combination with an higher contrast ratio and also would be great to have an option in settings to pick different colours. Also the blue is too light for some people and should be darker.

With this tool is easy to find a more accessible combination: https://webaim.org/resources/contrastchecker/
<img width="719" alt="image" src="https://user-images.githubusercontent.com/63522951/217861787-1452fcd3-9acd-40f6-99fc-b8418245ce01.png">
<img width="714" alt="image" src="https://user-images.githubusercontent.com/63522951/217862324-0a222d23-2ba8-4a89-af4d-65c1b371c45d.png">

Also, is not very clear which colour is what: apparently green for upload and blue for download.
Maybe adding arrows could help:
![image](https://user-images.githubusercontent.com/63522951/217860490-b8f78e66-7476-4966-8cfe-6c8bc8c3e717.png)

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.0 (280ace12f8)

**Comment by ilikepeaches** [comment]
see https://github.com/transmission/transmission/issues/4800

**Comment by ckerr** [comment]
This issue came first but 4800 is where all the discussion is, so :man_shrugging: I guess I'll close this one as a dupe of 4800.

@rebelcode-xr thanks for reporting this and thanks for the https://webaim.org/resources/contrastchecker/ link. I didn't know about that site and will be adding it to my toolchest :heart:

**Comment by rebelcode-xr** [comment]
happy to help 🌳
